pkg/apis/hive/v1/clusterdeployment_types.go

```diff
@@ -229,6 +229,11 @@ type ClusterDeploymentStatus struct {
 	// +optional
 	InstallerImage *string `json:"installerImage,omitempty"`
 
+	// ReleaseVersion is the version of OpenShift as reported by the release image
+	// resolved for the installation.
+	// +optional
+	ReleaseVersion *string `json:"releaseVersion,omitempty"`
```

Comment:
Is ReleaseVersion sufficiently differentiated from the clusters current version? Maybe "is the initial version of OpenShift that was installed, as reported..." would help a little.

Would InstallVersion or InstallReleaseVersion be better? InitialReleaseVersion?

Reply:
used `InstallVersion`

Reply:
@dgoodwin used `InstallVersion` to be clear that this version was reported for installation